src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java

```diff
@@ -1245,4 +1249,260 @@ public SBApiResponse workflowBulkUpdateTransitionV1(String userAuthToken, Multip
+		fields.add(Constants.USER_ID);
+		if (isMdoSearch) {
+			fields.add(Constants.ROOT_ORG_NAME);
+			fields.add(Constants.ROOT_ORG_ID);
+		} else {
+			filters.put(Constants.ROOT_ORG_ID, rootOrgId);
+			fields.add(Constants.PROFILE_DETAILS_PERSONAL_DETAILS_MOBILE);
+			fields.add(Constants.PROFILE_DETAILS_PERSONAL_DETAILS_FIRST_NAME);
+			fields.add(Constants.PROFILE_DETAILS_PERSONAL_DETAILS_PRIMARY_EMAIL);
+		}
+		Map<String, Object> request = this.getSearchObject(userIds, fields, filters);
+		HashMap<String, String> headersValue = new HashMap<>();
+		headersValue.put("Content-Type", "application/json");
+		Map<String, Object> allUserDetails = new HashMap<>();
+		StringBuilder url = new StringBuilder(configuration.getLmsServiceHost())
+				.append(configuration.getLmsUserSearchEndPoint());
+		Map<String, Object> searchProfileApiResp = (Map<String, Object>) requestServiceImpl.fetchResultUsingPost(url, request, Map.class, headersValue);
+		if (searchProfileApiResp != null
+				&& "OK".equalsIgnoreCase((String) searchProfileApiResp.get(Constants.RESPONSE_CODE))) {
+			Map<String, Object> map = (Map<String, Object>) searchProfileApiResp.get(Constants.RESULT);
+			Map<String, Object> response = (Map<String, Object>) map.get(Constants.RESPONSE);
+			List<Map<String, Object>> contents = (List<Map<String, Object>>) response.get(Constants.CONTENT);
+			if (!CollectionUtils.isEmpty(contents)) {
+				Map<String, Object> mdoDetails = new HashMap<>();
+				if (isMdoSearch) {
+					Map<String, Object> mdoDetailsResponse = contents.get(0);
+					mdoDetails.put(Constants.USER_ID, mdoDetailsResponse.get(Constants.USER_ID));
+					mdoDetails.put(Constants.ROOT_ORG_ID, mdoDetailsResponse.get(Constants.ROOT_ORG_ID));
+					mdoDetails.put(Constants.ROOT_ORG_NAME, mdoDetailsResponse.get(Constants.ROOT_ORG_NAME));
+					allUserDetails.put((String) mdoDetailsResponse.get("userId"), mdoDetails);
+				} else {
+					for (Map<String, Object> content : contents) {
+
+						HashMap<String, Object> profileDetails = (HashMap<String, Object>) content
+								.get(Constants.PROFILE_DETAILS);
+						if (!CollectionUtils.isEmpty(profileDetails)) {
+							HashMap<String, Object> personalDetails = (HashMap<String, Object>) profileDetails
+									.get(Constants.PERSONAL_DETAILS);
+							if (!CollectionUtils.isEmpty(personalDetails)) {
+								Map<String, Object> userPersonalInfo = new HashMap<>();
+								userPersonalInfo.put(Constants.FIRSTNAME, personalDetails.get(Constants.FIRSTNAME));
+								userPersonalInfo.put(Constants.MOBILE_NUMBER, personalDetails.get(Constants.MOBILE));
+								userPersonalInfo.put(Constants.PRIMARY_EMAIL, personalDetails.get(Constants.PRIMARY_EMAIL));
+								allUserDetails.put((String) content.get(Constants.USER_ID), userPersonalInfo);
+							}
+						}
+					}
+				}
+			}
+		}
+		return allUserDetails;
+	}
+
+	private void convertXSSFWorkbookToCSV(Workbook workbook, String csvFilePath) throws IOException{
```

Comment:
This is unnecessary... time and performance consuming... 

Do create a csv directly...

Reply:
This has been removed.